coverage.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -ex
+
+# Merge coverage reports from cypress and jest tests
+mkdir -p reports
+
+cp codecov-cypress/coverage-final.json reports/from-cypress.json
+cp codecov-jest/coverage-final.json reports/from-jest.json
```

Comment:
Doesnt this have to be something like
```
cp cypress-coverage/coverage-final.json reports/from-cypress.json
cp jest-coverage/coverage-final.json reports/from-jest.json
```

Reply:
I have verified that `codecov-jest`  [1] and `codecov-cypress` [2] is correct since the folders are explicitly defined as such.
[1] https://github.com/RedHatInsights/insights-remediations-frontend/pull/346/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R66
[2] https://github.com/RedHatInsights/insights-remediations-frontend/pull/346/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R138